NitroxClient/GameLogic/Vehicles.cs

```diff
@@ -69,7 +70,7 @@ public VehicleModel BuildVehicleModelFrom(GameObject gameObject, TechType techTy
                         opvehicle.Value.ReflectionCall("RegenerateRenderInfo", false, false);
                     }
 
-                    HSB = opvehicle.Value.subName?.GetColors();
+                    hsb = opvehicle.Value.subName.GetColors().ToDto();
```

Comment:
We might need to keep all of those safe navigation operator because that could lead to NRE,
because as I commented on my hacky fix for exosuit, SubName objects aren't always initialised yet at prematured spawn